app.py

```diff
@@ -125,33 +125,53 @@ def get_country_select_options(countries):
     return ["No data available"], ""
 
 
-@app.long_callback(
-    Output('education-content', 'children'),
-    Input('education-tabs', 'active_tab'),
-    running=[
-        (
-            Output("education-spinner", "style"),
-            {"display": "block"},
-            {"display": "none"},
-        ),
-        (
-            Output("education-content", "style"),
-            {"display": "none"},
-            {"display": "block"},
-        ),
-    ],
+@app.callback(
+    Output('stored-data-education', 'data'),
+    Input('stored-data-education', 'data')
```

Comment:
Since `dcc.Store(id='stored-data-education')` is declared in `education.py`, consider moving this callcack there?